src/pages/settings/Subscription/FreeTrialBadge.tsx

```diff
@@ -0,0 +1,44 @@
+import {useEffect, useState} from 'react';
```

Comment:
> Sorry, I thought I had commented.
> 
> I have checked all devices again and all the tests pass.
> 
> The PR is ready from side.

@abzokhattab Not sure why you removed `React` here. The app crashes due to this. Wondering why you are not getting the error?

![Screenshot 2024-09-16 at 10 22 05 AM](https://github.com/user-attachments/assets/c82168af-4138-488d-8f17-3e90e9aed343)

Reply:
Yeah i agree with Roji, its pretty quick and doesn't look like an issue to me either.